github/PullRequest.py

```diff
@@ -488,26 +496,34 @@ def is_merged(self):
         :calls: `GET /repos/:owner/:repo/pulls/:number/merge <http://developer.github.com/v3/pulls>`_
         :rtype: bool
         """
+        github._api_preview = True
```

Comment:
yeah, i didnt think about enterprise, i think we need to have a way to make it backwards compat.